packages/components/pie-modal/declaration.d.ts

```diff
@@ -7,3 +7,5 @@ declare module '*.scss?inline' {
     const content: Record<string, string>;
     export default content;
 }
+
+declare module '@justeattakeaway/pie-modal';
```

Comment:
I think that this is called a 'wildcard module declaration' and is sort of like defining the module, and its API's as `any`.

For the scope of your ticket I think this is absolutely fine, however I would love to help look into this further and see how we can improve these declarations (and if that is even necessary)

Reply:
Ah right, yes. It looks like `declaration.d.ts` is for declaring types for an otherwise JS module, and this shouldn't be necessary if we already have `index.d.ts` signposted in `package.json`. I'll try removing it again for now.